cmake/msys/msys_helper.sh

```diff
@@ -21,43 +21,43 @@ if [ $? -ne 0 ]; then
 fi
 
 info "Preparing the git directory..."
-mkdir $HOME/.git; touch $HOME/.git/config > /dev/null &2>1
+mkdir "$HOME"/.git; touch "$HOME"/.git/config > /dev/null &2>1
 git config --global http.sslverify false
 
 info "Cloning the repository..."
-if [ -d ./lmms ]; then 
+if [ -d ./lmms ]; then
 	warn "  - Skipping, ./lmms already exists"
-else 
+else
 	git clone -b $branch https://github.com/$fork/lmms.git
 fi
 
 
 info "Fetching ppa using cmake/msys/fetch_ppas.sh..."
-if [ -d $HOME/ppa ]; then
+if [ -d "$HOME"/ppa ]; then
         warn "  - Skipping, $HOME/ppa already exists"
 else
-	./lmms/cmake/msys/fetch_ppa.sh	
+	./lmms/cmake/msys/fetch_ppa.sh
 fi
 
 
 info "Extracting debs to $HOME/ppa/opt/, etc..."
-if [ -d $HOME/ppa/opt ]; then
+if [ -d "$HOME"/ppa/opt ]; then
         warn "  - Skipping, $HOME/ppa/opt already exists"
 else
         ./lmms/cmake/msys/extract_debs.sh
 fi
 
 info "Preparing library merge, making all qt headers writable..."
 chmod u+w /mingw64/include/qt4 -R
-chmod u+w /mingw32/include/qt4 -R 
+chmod u+w /mingw32/include/qt4 -R
 
 info "Merging mingw headers and libraries from ppa over existing system libraries..."
 find /mingw64 |grep sndfile.h
 
-if [ $? -ne 0 ]; then 
-	\cp -r $HOME/ppa/opt/mingw* /
-else 
-	warn "  - Skipping, sndfile.h has already been merged" 
+if [ $? -ne 0 ]; then
+	command cp -r "$HOME"/ppa/opt/mingw* /
```

Comment:
@zapashcanon this script is for Windows-only, so it won't be tested by Travis-CI.  I'm reading up on `\cp` in general and the above logic seems to be the proper replacement.  Apparently as a safeguard, the `cp` command is aliased to something like `cp -i` which is `interactive` mode.  Both `\cp` as well as `command cp` bypasses interactive mode, thus justifying the change.   In interactive terminal, `\cp` is valid, but in a script it appears to be both unnecessary as well as invalid.